qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -518,9 +518,10 @@ def _fetch_request(self, request, *, qnam=None, **kwargs):
         reply = qnam.get(request)
         return self.fetch(reply, **kwargs)
 
-    @pyqtSlot('QNetworkReply')
+    @pyqtSlot('QNetworkReply', result='QVariant')
```

Comment:
This is just [extra information for Qt/PyQt](https://www.riverbankcomputing.com/static/Docs/PyQt5/signals_slots.html#the-pyqtslot-decorator) and has nothing to do with type hints. It's only relevant for QML which we don't use, so I don't see why it should be passed here.

Reply:
I was not sure how to tell mypy that the result type is not `None`...

Reply:
Via the `-> DownloadItem` annotation you already have below?

Reply:
But mypy on Travis gives error...

Reply:
The error I get locally is:

```
qutebrowser/browser/qtnetworkdownloads.py:519: error: Incompatible return value
type (got "None", expected "DownloadItem")  [return-value]
            return self.fetch(reply, **kwargs)
```

I fixed that one half a year ago in https://github.com/stlehmann/PyQt5-stubs/commit/eeadc4cafe65118ccf7e7df8edbd02bb52b58010 but I'm still using my own `qutebrowser/PyQt5-stubs` fork for this repository because I haven't upstreamed all my fixes...

I opened #5368 to switch back to the upstream thing - for now, remove the `result=`, and where mypy complains about a return value being `None`, add `# type: ignore`.